### PR TITLE
[risk=no] Enable Fitbit config flags through preprod

### PR DESCRIPTION
**Pull request by dolbeew**

### CODE REVIEW

File: api/config/config_staging.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
     "enableReportingUploadCron": true,
     "enableCOPESurvey": false,
```
Comment (freemabd) [inline api/config/config_staging.json:113]: Please set this to true

File: api/config/config_stable.json [hunk 2]
```diff
@@ -112,7 +112,7 @@
     "enableReportingUploadCron": true,
     "enableCOPESurvey": false,
```
Comment (freemabd) [inline api/config/config_stable.json:113]: Please set this to true

**Review by freemabd — APPROVED** [review]
Looks good.. Please turn on all flags thru preprod

**Comment by dolbeew** [comment]
Done. Also enabled both in perf env

**Pull request merged**